src/common/itt/CMakeLists.txt

```diff
@@ -12,8 +12,12 @@ add_library(openvino::itt ALIAS ${TARGET_NAME})
 target_link_libraries(${TARGET_NAME} PUBLIC openvino::util)
 
 if(ENABLE_PROFILING_ITT)
-    target_link_libraries(${TARGET_NAME} PUBLIC
```

Comment:
Why modified? Was previous version not working?

Reply:
yes, it does not work well with aliased targets.